src/main/java/ua/mateacademy/tania/lesson14/rest/MateGroupServiceImpl.java

```diff
@@ -76,4 +78,77 @@ public Response removeStudent(@PathParam("groupId")int groupId, @PathParam("surn
         mateGroup.getStudents().removeIf(student -> student.getSurname().equals(surname));
         return Response.status(ACCEPTED).entity(mateGroup).type(MediaType.APPLICATION_JSON).build();
     }
+
+    @Override
+    @PUT
+    @Path("/humanResource")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
why do you use these annotations here?

Reply:
> why do you use these annotations here?

because of i need to pass data(HumanResource) and then to get response with entity